smokeTest.js

```diff
@@ -0,0 +1,83 @@
+  describe('001: Smoke test ', function() {
+    
+    browser.waitForAngularEnabled(false);
+    browser.ignoreSynchronization = true;
+    browser.manage().window().maximize();
+    browser.get('http://automationpractice.com/index.php');
+    var EC = protractor.ExpectedConditions;
+
+    it('001: User is able to sign in, search items, view item images, choose quantities and sizes, add it to cart, choose paying method and complete order', function() {
```

Comment:
Add async/await or method chaining with .then() for better control of test steps execution

https://medium.com/@anandujjwal/end-to-end-testing-using-protractor-with-async-await-46dc11d948d0

https://www.toolsqa.com/protractor/promise-protractor/#:~:text=It%20also%20helps%20automation%20tests,function%20to%20handle%20the%20result